src/components/Header/index.js

```diff
@@ -1,48 +1,160 @@
+            </Col>
+          </Row>
+          <Row className="toggle-container"type="flex" align="middle"> 
+            {user.username && 
+            <Col>
+              <Link to="/new"><Icon  style={{fontSize: '22px'}} type="edit"  /></Link>
+            </Col>
+            }
+            {!user.username && <a href={this.getOathURL()}><Button>Login</Button></a>}
+            {user.username &&
+            <Col className="ml">
+              <span className="mx-auto ml" style={{ marginRight: 24 }}>
+                <Badge count={1} style={{ fontSize: '14px'}} ><Avatar style={{backgroundColor: '#22419c', fontSize: '14px'}}  shape="square" icon="bell" /></Badge>
+              </span>
+            </Col>
+            }  
+            <Col>
+              {user.username &&
+              <Col className="ml" >
+                <div >
+                  <Dropdown overlay={menu}>
+                    <a className="ant-dropdown-link" >
+                      <Avatar size="small" src={`https://steemitimages.com/u/${user.username}/avatar`}  icon="user" />
+                    </a>
+                  </Dropdown>
+                </div>
+              </Col>
+              } 
+            </Col>
+            <Col className="navbar__link-toggle">
+              <Icon onClick={e => this.handleHeaderToggle(e)} type="menu-fold" style={{fontSize: '20px'}} />
+            </Col>
+          </Row> 
+        </Row>
+        <Row type="flex" justify="center" align="middle" style={Object.assign({}, styles.header)} className="collasped-header">
+          <Row type="flex" align="middle">
+            <Col  className=" mb">
+              <KnackSelect/>
+            </Col>
+          </Row>
+          <Row type="flex" align="middle">
+            <Col className=" collasped-search mb">
+              <KnackSearch onSearch={(value) => {this.setState({searchString: value});}}/>
+            </Col>
+          </Row>
+        </Row>
+      </Header>
+    );
+  } 
+
+  getOathURL () {
+    return SteemConnect.getLoginURL();
+  }
+
+}
+
+const mapStateToProps = state => {
+  return {
+    user: state.user,
+    articles: state.articles,
+    header: state.header
   };
-  return (
-    <Header>
-      <div id="header-wrapper">
-        <div className="logo"><img src={logo} alt="Knacksteem Logo" /></div>
-        <div className="username-login">
-          {user.username && <Link to="/new"><Button type="primary">New Contribution</Button></Link>}
-          {!user.username && <a href={getOathURL()}><Button>Login</Button></a>}
-          {user.username && <Link to="/"><Button onClick={onLogoutClick}>Logout</Button></Link>}
-          {user.username && <Tooltip title={user.username}><a href={`https://steemit.com/@${user.username}`}><div className="avatar" style={{backgroundImage: `url(https://steemitimages.com/u/${user.username}/avatar)`}} /></a></Tooltip>}
-        </div>
-      </div>
-    </Header>
-  );
+
+=======
```

Comment:
This got left while resolving conflicts

Reply:
We have to get rid of that Header component
Created a new header component called KnackHeader